waspc/data/Cli/templates/basic/src/client/MainPage.jsx

```diff
@@ -1,4 +1,4 @@
-import waspLogo from './waspLogo.png'
+import waspLogo from './waspLogo.svg'
```

Comment:
This is really one of the most important changes, this change here.
This tells Wasp to, when it generates new project, use .svg in it.
But that waspLogo.svg, you need to provide it also somewhere in the `data/Cli/templates/basic/src/client/` dir, I don't think you did though, I don't think you replaced that waspLogo.png with waspLogo.svg? You should do that.